### ISSUE TITLE
Turn down k8s-gubernator/g8r cluter

### ISSUE THREAD

**Issue by spiffxp** [kubernetes/k8s.io#789]
We can do this once:
- [ ] kettle has been migrated (#787)
- [ ] spiffxp@ has confirmed there's really nothing else on this cluster

/wg k8s-infra
/area cluster-mgmt
/assign @spiffxp

**Comment by BenTheElder** [comment]
cc @mushuee

**Comment by BenTheElder** [comment]
/remove-lifecycle stale
/lifecycle frozen
blocked on https://github.com/kubernetes/k8s.io/issues/787

**Comment by BenTheElder** [comment]
Yeah, I've confirmed that there's no more spend in there for the past two weeks so that should be proof enough that we've actually turned everything down.
